src/Client/Web/Bit.Client.Web.BlazorUI/Components/Pivot/BitPivot.razor

```diff
@@ -0,0 +1,35 @@
+﻿@namespace Bit.Client.Web.BlazorUI
+@inherits BitComponentBase
+
+<div @ref="RootElement"
+     style="@StyleBuilder.Value"
+     class="@ClassBuilder.Value">
+    <CascadingValue Value="this"
+                    IsFixed="true">
+        <div>
+            @foreach (var item in Items)
+            {
+                <div @onclick="() => ItemClicked(item)"
+                     class="@(Items[SelectedKey] == item.Value ? "selected-item" : string.Empty)"
+                     style="@(item.Value.Visibility == ComponentVisibility.Collapsed ? "display:none" : item.Value.Visibility == ComponentVisibility.Hidden ? "visibility:hidden" : string.Empty)">
```

Comment:
again, please create a method in codebehind and move this logic in it

Reply:
moved logic to codebehind